src/link/tests/ipvtap.rs

```diff
@@ -16,7 +16,7 @@ fn test_ipvtap_link_info() {
         0x00, 0x00, 0x00, 0x00, 0x24, 0x00, 0x12, 0x00, 0x0b, 0x00, 0x01, 0x00,
         0x69, 0x70, 0x76, 0x74, 0x61, 0x70, 0x00, 0x00, 0x14, 0x00, 0x02, 0x00,
         0x06, 0x00, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x06, 0x00, 0x02, 0x00,
-        0x02, 0x00, 0x00, 0x00,
+        0x01, 0x00, 0x00, 0x00,
```

Comment:
Please do not modify captured data.

If you want to test your enum value, please create new test case and do the nl_mon capture by following the README file of this repo.